### ISSUE TITLE
Feature request: venue = "html"

### ISSUE THREAD

**Issue by cwickham** [tidyverse/reprex#236]
A few places I would like to post reprexes, most notably the LMS [canvas](https://canvas.instructure.com/), won't accept markdown, but will allow HTML input.  

I propose an additional `venue = "html"` that would render using `output: html_fragment`.  So, for example the code:
```r
reprex({
  (y <- 1:4)
  mean(y)
}, venue = "html")
```
would place the following on the clipboard:
```html
<pre class="r"><code>(y &lt;- 1:4)
#&gt; [1] 1 2 3 4
mean(y)
#&gt; [1] 2.5</code></pre>
```

This can be pasted into the "HTML Editor" in canvas:
<img width="867" alt="screen shot 2019-01-21 at 7 21 42 pm" src="https://user-images.githubusercontent.com/25964/51510807-7f15d700-1db3-11e9-8d7f-bdfd019c29ac.png">

Which canvas renders into something like:
<img width="882" alt="screen shot 2019-01-21 at 7 22 16 pm" src="https://user-images.githubusercontent.com/25964/51510785-64436280-1db3-11e9-8c7e-7ced49159519.png">

I have an attempt implementing this which only involves an additional file to `make_filenames()` and an additional condition in `reprex()`.  Would you accept a PR for this?  

Any thoughts on any lurking gotchas with this venue?

**Comment by jennybc** [comment]
I think this is a good idea. I've considered it before! We are generally making html anyway for preview purposes.

**Comment by jennybc** [comment]
Closed via d03e0ef8105d6e505e97cd8d0a46e73d7a8676b2
